contracts/starknet/objects/vote.cairo

```diff
@@ -0,0 +1,9 @@
+%lang starknet
+
+from contracts.starknet.lib.types import EthAddress
```

Comment:
We should consolidate the types either into the types.cairo file or the objects directory. I dont mind which way

Reply:
Both are good